Source/components/Override.js

```diff
@@ -0,0 +1,54 @@
+import { DefinedPropertiesKey, WrappedObject } from '../WrappedObject'
+import { Types } from '../enums'
+import { Factory } from '../Factory'
+
+/**
+ * An MSAvailableOverride. This is not exposed, only used by SymbolInstance
+ */
+export class Override extends WrappedObject {}
+
+Override.type = Types.Override
+Override[DefinedPropertiesKey] = { ...WrappedObject[DefinedPropertiesKey] }
+Factory.registerClass(Override, MSAvailableOverride)
+
+Override.define('path', {
```

Comment:
Would it be better to call this `layerID`?

It looks like it's only used in the tests, but it feels like it could be a point of confusion. If you had a nested text layer the name could be `symbolID/textLayerID_stringValue`. But this `path` is just going to return `textLayerID`.

Likewise, I'd be tempted to rename `id` as `name` - although I don't think that's as confusing.

Reply:
Oh I want to return `symbolID/textLayerID` here. Haven't looked into nested symbols yet.

For the `id`, it's because I need an `id` property since the SymbolInstance inherits from Layer which defines an Id. Hopefully it's just a unique identifier and people won't need to use it

Reply:
> Oh I want to return symbolID/textLayerID here.

There's a private method on `MSOverridePoint` called `layerIDPath` that would give you want you want.